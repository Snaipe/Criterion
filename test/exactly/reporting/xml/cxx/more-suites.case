[act]

% more-suites.cc.bin --xml

[assert]

stdout
  -transformed-by ( replace_version | replace_times )
  equals -contents-of expected/more-suites.txt
