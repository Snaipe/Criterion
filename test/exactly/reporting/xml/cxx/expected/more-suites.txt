<?xml version="1.0" encoding="UTF-8"?>
<!-- Tests compiled with Criterion <VERSION> -->
<testsuites name="Criterion Tests" tests="3" failures="0" errors="0" disabled="1">
  <testsuite name="suite2" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="<TIME>">
    <testcase name="test" assertions="1" status="PASSED" time="<TIME>">
    </testcase>
  </testsuite>
  <testsuite name="suite1" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="<TIME>">
    <testcase name="test" assertions="1" status="PASSED" time="<TIME>">
    </testcase>
  </testsuite>
  <testsuite name="disabled" tests="1" failures="0" errors="0" disabled="1" skipped="1" time="<TIME>">
    <testcase name="test" assertions="0" status="SKIPPED" time="<TIME>">
      <skipped/>
    </testcase>
  </testsuite>
</testsuites>
